src/lxc/log.h

```diff
@@ -287,7 +288,9 @@ ATTR_UNUSED static inline void LXC_##LEVEL(struct lxc_log_locinfo* locinfo,	\
 		 * without restrictions. So let's use it for our		\
 		 * logging stamps.						\
 		 */								\
+		saved_errno = errno;						\
 		(void)clock_gettime(CLOCK_REALTIME, &evt.timestamp);		\
+		errno = saved_errno;						\
```

Comment:
any reason not to restore after the `va_end`?
I think that might make half the other related commits superfluous? Maybe?

Reply:
Good call! Done!